go.mod

```diff
@@ -7,13 +7,13 @@ require (
 	github.com/aws/aws-sdk-go v1.35.35
 	github.com/dylanmei/iso8601 v0.1.0
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/giantswarm/apiextensions/v3 v3.8.0
+	github.com/giantswarm/apiextensions/v3 v3.9.0
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/badnodedetector v1.0.1
 	github.com/giantswarm/certs/v3 v3.1.0
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient/v5 v5.0.0
-	github.com/giantswarm/k8scloudconfig/v9 v9.1.3
+	github.com/giantswarm/k8scloudconfig/v9 v9.2.0
```

Comment:
This is the only change I really wanted, everything else was bumped by go modules automagically.